src/app/controllers/CustomerController.js

```diff
@@ -248,7 +248,7 @@ class CustomerController {
     sendPasswordResetCode(email_address, method = 'EMAIL') {
         return new Promise((resolve, reject) => {
             let mutationString = `
-                mutation sendCustomerPasswordResetCode ($email_address: String!, $method:CustomerPasswordResetCodeSendMethod) {
+                mutation sendCustomerPasswordResetCode ($email_address: String!, $method:ResetCodeSendMethod) {
```

Comment:
```suggestion
                mutation sendCustomerPasswordResetCode ($email_address: String!, $method: ResetCodeSendMethod) {
```
Please add spaces lol